src/relayserver/RelayServer.ts

```diff
@@ -701,7 +721,11 @@ latestBlock timestamp   | ${latestBlock.timestamp}
   }
 
   _isTrustedPaymaster (paymaster: string): boolean {
-    return this.trustedPaymastersGasLimits.get(paymaster.toLocaleLowerCase()) != null
+    return this.trustedPaymastersGasAndDataLimits.get(paymaster.toLocaleLowerCase()) != null
```

Comment:
toLowerCase. (not sure why the "locale" version was used here)

Reply:
Not Part of this PR, not sure who put `Locale` there